src/App.tsx

```diff
@@ -0,0 +1,200 @@
+import "./App.css";
+import React from "react";
+import { Accordion, Panel } from "baseui/accordion";
+import { Button, KIND, SHAPE } from "baseui/button";
+import { Card, StyledBody, StyledAction } from "baseui/card";
+import { Checkbox, LABEL_PLACEMENT } from "baseui/checkbox";
+import { FormControl } from "baseui/form-control";
+import { Input } from "baseui/input";
+import { Slider } from "baseui/slider";
+import { useStyletron } from "baseui";
+
+function App() {
+  const [css, theme] = useStyletron();
+
+  const [password, setPassword] = React.useState<string>("");
+  const passwordRef = React.useRef(null);
```

Comment:
This looks redundant to me, I don't see where this ref is being used. 
Remove if un-necessary pls